### PR TITLE
Multi-Layer GCN model s4640623 Marco Shenyang Day

### PR DESCRIPTION
**Pull request by MarcoShenyang**
I have made a multi-layer GCN model and used the Facebook Large Page-Page Network dataset for training. During training, I achieved a loss of  0.2661 and accuracy of 0.9136. However; while testing the model I achieved a loss of 1.5884 and accuracy of 0.2665 showing that I have overfitting occurring.

### CODE REVIEW

**Comment by gayanku** [comment]
GCN code looks good and train accuracy seems fine. No TSNE or Test/Val accuracy provided. Predict uses an untrained model and no provision to save and load a pre-trained model is provided.

**Comment by shakes76** [comment]
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate use and implementation (no saving model, prediction not properly implemented, no testing accuracy) -4
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem (No UMAP or TSNE) -3
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage (no maps etc.) -2
Module present
Commenting
No Data leakage, but not validation or testing data used -2
Difficulty: Normal -5
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
ReadMe minimal, need more info -2
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Feedback required, but will not merge, can't tell if problem solved properly.
Request Description minimal, need more info -1
